extensions/mssql/src/utils.ts

```diff
@@ -304,3 +306,31 @@ export async function exists(path: string): Promise<boolean> {
 		return false;
 	}
 }
+
+export async function getOrDownloadServer(config: IConfig, handleServerEvent?: (e: string, ...args: any[]) => void): Promise<string> {
+	// This env var is used to override the base install location of STS - primarily to be used for debugging scenarios.
+	try {
+		const stsRootPath = env['ADS_SQLTOOLSSERVICE'];
```

Comment:
In general, I like using an env var here, but I'm a little concerned that, for signoffs, folks can forget that this path is set and be testing different bits than they're expecting.

Can we make this a little more clear that bits are being loaded (console logs can be missed -- maybe an info message?), or restrict this to dev builds only?

Reply:
Yeah, that's a good idea. I initially dismissed the idea of having a popup for this to avoid too much spam but given how infrequently this would be used and the issue you mentioned it seems reasonable to do that.